content/articles/proppy-release.md

```diff
@@ -0,0 +1,23 @@
+Title: Proppy is officially live!
+Date: 2016-10-11
+Short_summary: Our product is out of beta, check it out
+Category: Product
+Authors: Vincent
+
+*Reading time: ~1 minute.*
+
+[Back in March](https://blog.wearewizards.io/we-are-launching-our-first-product) we announced [Proppy](https://proppy.io/), our tool to write beautiful proposals.
+
+With all the excitement after releasing Proppy last week, we actually forgot to blog about it!
+<!-- PELICAN_END_SUMMARY -->
+
+After more than 6 months of constant improvements in beta and over 1500 signups, we have decided that Proppy is at the point where it is a tool worth paying for.
+
+Some larger features like analytics and integrations are still missing but they are actively being worked on and should land soon (one could say imminently for analytics).
+
+In order to turn Proppy into a sustainable business (ie. where we don't have to take external contracts like right now), we will need to spend time on marketing.
+Since this is a new thing for us, we appreciate any advice you may have.
```

Comment:
"This is our first real product with support, proper invoicing & design, so we appreciate any and all advice you can share!" maybe?